src/libs/antares/study/parameters/adq-patch-params.h

```diff
@@ -59,7 +59,7 @@ enum class AdqPatchPTO
 
 struct LocalMatching
```

Comment:
`struct LocalMatching` : do we keep this **struct** to have a minimal change ?

Reply:
yes